sdk/test-utils/recorder/package.json

```diff
@@ -84,9 +87,13 @@
     "@types/node": "^12.0.0",
     "@types/mock-require": "~2.0.0",
```

Comment:
could we double check the list of deps and take out any unused ones? For example, do we still need `@types/nise`?

Reply:
good point, removed a ton of packages